dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -37,17 +37,48 @@ warning.could.not.thin.application=CWWKC0251W: Could not convert the application
+#UPDATED for SpringBoot 3.0:
+# error.need.springboot.version.15=CWWKC0252E: The application failed to start because the springBoot-2.0 feature is configured in the server.xml file. The application requires the springBoot-1.5 feature to be configured.
+#error.need.springboot.version.15.explanation=The application requires the springBoot-1.5 feature. Check the version of the Spring Boot feature that is currently configured in the server.xml file.  
+#error.need.springboot.version.15.useraction=Configure the springBoot-1.5 feature in the server.xml file. Remove any other Spring Boot features.
+
+#0: The required spring feature.
+#1: The current spring feature.
+#2: The version of spring content found in the application.
+error.spring.required.15=CWWKC0252E: The {0} feature must be configured in the server.xml file. The {1} feature  is configured and the application has Spring {2} content.
+error.spring.required.15.explanation=The currently configured Spring Boot feature is not compatible with application content.
+error.spring.required.15.useraction=Configure a version of the Spring Boot feature that is compatible with application content.
+
+#UPDATED for SpringBoot 3.0:
+#error.need.springboot.version.20=CWWKC0253E: The application failed to start because the springBoot-1.5 feature is configured in the server.xml file. The application requires the springBoot-2.0 feature to be configured.
+#error.need.springboot.version.20.explanation=The application requires the springBoot-2.0 feature. Check the version of the Spring Boot feature that is currently configured in the server.xml file.
+#error.need.springboot.version.20.useraction=Configure the springBoot-2.0 feature in the server.xml file. Remove any other Spring Boot features.
+
+#0: The required spring feature.
+#1: The current spring feature.
+#2: The version of spring content found in the application.
+error.spring.required.20=CWWKC0253E: The {0} feature must be configured in the server.xml file. The {1} feature is configured and the application has Spring {2} content.
+error.spring.required.20.explanation=The currently configured Spring Boot feature is not compatible with application content.
+error.spring.required.20.useraction=Configure a version of the Spring Boot feature that is compatible with application content.
+
+#ADDED for SpringBoot 3.0:
+error.spring3.required=CWWKC0273E: Feature springBoot-3.0 must be configured: Feature springBoot-1.5 or springBoot-2.0 is configured and the application has Spring 3.0 content.
+error.spring3.required.explanation=The currently configured Spring Boot feature is not compatible with application content.
+error.spring3.required.useraction=Configure a version of the Spring Boot feature that is compatible with application content.
+
+#UPDATED for SpringBoot 3.0:
+#error.missing.servlet=CWWKC0254E: The application failed to start because it requires a servlet feature to be configured in the server.xml file.
```

Comment:
```suggestion
#error.missing.servlet=CWWKC0254E: The application failed to start because it requires a Servlet feature to be configured in the server.xml file.
```